lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -596,17 +596,62 @@ std::string SerializedModuleBaseName::getName(file_types::ID fileTy) const {
   return std::string(result.str());
 }
 
+llvm::Optional<std::string> SerializedModuleBaseName::getPackageInterfacePathIfInSamePackage(llvm::vfs::FileSystem &fs, ASTContext &ctx) const {
```

Comment:
nitpick: This line looks unusually long, can we break it apart to try to keep in under the ~80 char per line limit?